app/src/main/java/com/cornellappdev/uplift/ui/components/Home/HomeCard.kt

```diff
@@ -0,0 +1,128 @@
+
+
+@Composable
+    fun homeCard(gymexample:Gym ) {
+    var paintres = remember  {mutableStateOf(if(gymexample.isFavorite()) R.drawable.ic_star_filled else R.drawable.ic_star) }
+        Box(modifier= Modifier
+            .padding(5.dp)
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .aspectRatio(2F / 1F)){
+            Card(
+                modifier = Modifier
+            ) {
+                Column() {
+                    Column(modifier = Modifier.weight(3F)) {
+                        Image(
+                            modifier = Modifier.fillMaxWidth(),
+                            painter = painterResource(id=R.drawable.court_lines),
+                            contentDescription = "Gym Image",
+                            contentScale = ContentScale.FillWidth
+                        )
+
+                    }
+                    Column(modifier = Modifier.weight(2F)) {
+                        Row(
+                            modifier = Modifier.padding(5.dp)
+                        ) {
+                            Text(
+                                text =gymexample.name,
+                                fontSize = 20.sp,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.weight(1F))
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_dumbbell),
+                                contentDescription = "Dumbell",
+                            )
+                            Image(
+                                painter = painterResource(id = R.drawable.ic_bowling_pins),
+                                contentDescription = "Bowling Pins",
+                            )
+
+                        }
+                        Row(modifier = Modifier.padding(start = 5.dp, end = 5.dp)) {
+                            Text(
+                                text = "Open",
+                                fontSize = 12.sp,
+                                color = Color.Green,
+                                fontFamily = montserratFamily
+
+                                )
+                            Spacer(modifier = Modifier.width(10.dp))
+                            Text(
+                                text = "Closes at 6 pm",
+                                fontSize = 12.sp,
+                                fontFamily = montserratFamily
+                                )
+
+                        }
+                        Row(
+                            modifier = Modifier.padding(
+                                start = 5.dp,
+                                end = 5.dp
+                            )
+                        ) {
+                            Text(
+                                text = "Cramped",
+                                fontSize = 12.sp,
+                                color = Color.Red,
```

Comment:
I noticed you used a red color instead of the orange designated on the Figma. Of course, this isn't that big of a deal right now, since we don't need to implement gym capacity right now... but for now, we should probably still import the orange color. Please do so in `Colors.kt` by entering a color as follows:

`val COLOR_NAME = Color(0xFF[HEX CODE HERE])`